tutorials/hyperspectral/Plot-Hyperspectral-Spectra.Rmd

```diff
@@ -55,7 +55,7 @@ before moving on to this tutorial.
 
 
 In this tutorial, we will extract a single-pixel's worth of reflectance values to
-plot a spectral profile for that pixel.
+plot a spectral signature for that pixel.
 
```

Comment:
Can you add slightly more commentary into this tutorial about why steps are happening.  Even here, something like,  "We may want to do this for XX reason.  
We need to start by setting our working directory and loading our data".

Reply:
Certainly. I added quite a bit of explanation to the introduction, and throughout the tutorial to better describe the process from step to step.